libnavui-base/src/main/java/com/mapbox/navigation/ui/base/api/voice/SpeechApi.kt

```diff
@@ -9,9 +9,12 @@ import com.mapbox.navigation.ui.base.model.voice.Announcement
 interface SpeechApi {
 
     /**
-     * Given [VoiceInstructions] the method will generate the voice instruction [Announcement].
+     * Given [VoiceInstructions] the method will try to generate the
+     * voice instruction [Announcement] including the synthesized speech mp3 file
+     * from Mapbox's API Voice.
```

Comment:
```suggestion
     * from Mapbox's API Voice.
     *
     * In case of an error, a fallback is provided that can be played with a text-to-speech engine.
     *
```